packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -188,8 +188,7 @@ class RunCommand extends RunCommandBase {
       )
       ..addFlag('fast-start',
```

Comment:
Is the longer term plan to remove this flag or does it still make sense after this PR?